examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
<p>"an own dataset" sounds odd. Suggest changing to "your own dataset". But this is still strange, as movielens is just a public dataset.  Maybe just: "Case study: Applying NVTabular and Merlin model to the movielens data"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>